plugins/adr/README.md

```diff
@@ -71,16 +71,32 @@ Afterwards, add the following code snippet to use `AdrSearchResultListItem` when
 import { AdrSearchResultListItem } from '@backstage/plugin-adr';
 
 ...
+<SearchType.Accordion
+  name="Result Type"
+  defaultValue="software-catalog"
+  types={[
+    ...
+    {
+      value: 'adrs',
+      name: 'Architecture Decision Records',
+      icon: <DocsIcon />,
+    },
+  ]}
+/>
 
-case 'adr':
+...
+
+case 'adrs':
```

Comment:
is the previous `case 'adr'` a typo that should have been adrs from the beginning?

Reply:
This seems like it was correct before: https://github.com/backstage/backstage/blob/f0ca6e0adf4566097f58484df21df65b2ab564c3/plugins/adr-backend/src/search/DefaultAdrCollatorFactory.ts#L99